pkg/cli/admin/release/info.go

```diff
@@ -735,15 +741,28 @@ func (i *ReleaseInfo) Platform() string {
 }
 
 func (o *InfoOptions) LoadReleaseInfo(image string, retrieveImages bool) (*ReleaseInfo, error) {
+	replacedRef := false
+	opts := extract.NewExtractOptions(genericclioptions.IOStreams{Out: o.Out, ErrOut: o.ErrOut})
+	opts.SecurityOptions = o.SecurityOptions
+	opts.FileDir = o.FileDir
 	ref, err := imagesource.ParseReference(image)
 	if err != nil {
 		return nil, err
 	}
+	fromContext, err := opts.SecurityOptions.Context()
+	if err != nil {
+		return nil, err
+	}
+	fromContext.AlternativeSources = &opts.SecurityOptions
```

Comment:
This looks off, why you're assigning `opts.SecurityOptions` to a context which was just retrieved from `opt.SecurityOptions` a few lines above?

Reply:
I was expecting that all of alternative sources is done through `WithICSP` you've introduced in library-go.

Reply:
moved that all to library-go, updated